acme/account.go

```diff
@@ -20,6 +20,15 @@ type Account struct {
 	Status                 Status           `json:"status"`
 	OrdersURL              string           `json:"orders"`
 	ExternalAccountBinding interface{}      `json:"externalAccountBinding,omitempty"`
+	LocationPrefix         string           `json:"-"`
+}
+
+// GetLocation returns the URL location of the given account.
+func (a *Account) GetLocation() string {
+	if a.LocationPrefix == "" {
+		return ""
+	}
+	return a.LocationPrefix + a.ID
 }
```

Comment:
Can we add a test that ensures that the location prefix generated below, using `getAccountLocationPath(ctx, linker, "")`, always ends with `/` so this URL is properly formed?